src/myrtlespeech/protos/rnn_t_encoder.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package myrtlespeech.protos;
+
+import "myrtlespeech/protos/rnn.proto";
+import "myrtlespeech/protos/fully_connected.proto";
+
+// RNN-T Encoder configuration
+message RNNTEncoder {
```

Comment:
I have renamed to TransducerEncoder on the basis that this class could, in theory, be generalised so that it doesn't require RNN module. Does this work?